website/extensions/blockquote.md

```diff
@@ -0,0 +1,70 @@
+# Blockquote
+
+The `Blockquote` node represents a block of quoted text. It can be used to create a block of text that is quoted from another source.
```

Comment:
```suggestion
The `blockquote` node represents a block of quoted text. It can be used to create a block of text that is quoted from another source.
```